packages/client/src/helpers/FetchGraphQLClient.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLClient } from 'graphql-request';
+
+type ClientOptions = ConstructorParameters<typeof GraphQLClient>[1];
+type Options = Omit<ClientOptions, 'fetch'>;
```

Comment:
fixed as requested in https://github.com/lens-protocol/lens-sdk/pull/287#discussion_r1162620656